bin/kano-track-space

```diff
@@ -0,0 +1,73 @@
+#!/usr/bin/env python
+#
+# Copyright (C) 2018 Kano Computing Ltd.
+# License: http://www.gnu.org/licenses/gpl-2.0.txt GNU GPL v2
+#
+# kano-os-loader
+#
+# We install this script via cmdline.txt "init" kernel parameter
+#
+
+import subprocess
+from itertools import imap
+import os
+from kano_profile.tracker import track_data
+
+
+def get_package_space():
+    # return space used by debian packages (in units of bytes, but not
+    # expected to be precise to 1 byte)
+    o = subprocess.Popen("dpkg-query -Wf '${Installed-Size}\n'",
```

Comment:
You could use `kano.utils.shell.run_cmd` for this here.

Reply:
That runs ﻿`﻿process.communicate()` and hence reads all of the output into memory, which I wanted to avoid here.